plugins/pay.c

```diff
@@ -1833,7 +1833,8 @@ static void init(struct plugin *p,
 	maxdelay_default = atoi(field);
 }
 
-struct payment_modifier *paymod_mods[3] = {
+struct payment_modifier *paymod_mods[4] = {
```

Comment:
Still don't need `[4]` here: `[]` is sufficient and correct.

Reply:
Went through the commits and removed all explicit size arguments.